### ISSUE TITLE
False positive finding for USE_DATA_CLASS rule on @Stable annotated class

### ISSUE THREAD

**Issue by WebTiger89** [saveourtool/diktat#1316]
@Stable is an android/Jetpack Compose construct

The suspicious code snippet:
```
@Stable
class PermissionState {
    var showRationale by mutableStateOf(false)
}
```

**Comment by petertrr** [comment]
I think it's another case #802. We should definitely support a configurable set of excludes for this rule.
